adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -909,6 +914,14 @@ private void setPaypalEcsData(PaymentRequest paymentRequest, CartData cartData)
         paymentRequest.setPaymentToken(cartData.getAdyenPaymentToken());
     }
 
+    private void setSepaDirectDebitData(PaymentsRequest paymentRequest, CartData cartData) {
+        DefaultPaymentMethodDetails paymentMethodDetails = new DefaultPaymentMethodDetails();
```

Comment:
should we use the new `SepaDirectDebitDetails` class?
https://github.com/Adyen/adyen-java-api-library/blob/7b713a5bee161e9ddc1c5962d2c9b44795196f15/src/test/java/com/adyen/CheckoutTest.java#L443

Reply:
This is supported in latest java library and hybris is on older version so this class does not exist. But it is a good point for future release when we update library version in Hybris.